supabase/migrations/20250311000000_update_alert_view.sql

```diff
@@ -0,0 +1,16 @@
+-- If a tenant has `external` payment method then we do not
+-- want to trigger any messaging about missing payment.
+
+CREATE OR REPLACE VIEW internal.alert_free_trial_ending AS
+ SELECT 'free_trial_ending'::public.alert_type AS alert_type,
+    (((tenants.tenant)::text || 'alerts/free_trial_ending'::text))::public.catalog_name AS catalog_name,
+    json_build_object('tenant', tenants.tenant, 'recipients', array_agg(DISTINCT jsonb_build_object('email', alert_subscriptions.email, 'full_name', (users.raw_user_meta_data ->> 'full_name'::text))), 'trial_start', tenants.trial_start, 'trial_end', ((tenants.trial_start + '1 mon'::interval))::date, 'has_credit_card', bool_or((customers."invoice_settings/default_payment_method" IS NOT NULL) OR (tenants.payment_provider != 'external'))) AS arguments,
```

Comment:
This is like the previous alert update. This migration is taken by copying what is in the `` and then adding what we need.

The actual change is adding `OR (tenants.payment_provider != 'external')` to the `has_credit_card` argument.

Reply:
Shouldn't this actually be `OR (tenants.payment_provider == 'external') to the has_credit_card`, with `==` instead of `!=`?

Reply:
fixed!